packages/@okta/i18n/package.json

```diff
@@ -1,101 +1,38 @@
 {
   "_args": [
     [
-      "@okta/i18n@37.2.0",
+      {
+        "name": "@okta/i18n",
+        "raw": "@okta/i18n@37.3.0",
+        "rawSpec": "37.3.0",
+        "scope": "@okta",
+        "spec": "37.3.0",
+        "type": "version"
+      },
       "/home/vagrant/okta/okta-signin-widget"
     ]
   ],
-  "_from": "@okta/i18n@37.2.0",
-  "_id": "@okta/i18n@37.2.0",
+  "_from": "@okta/i18n@37.3.0",
+  "_id": "@okta/i18n@37.3.0",
   "_inCache": true,
   "_installable": true,
   "_location": "/@okta/i18n",
-  "_phantomChildren": {
-    "abbrev": "1.1.1",
-    "argparse": "1.0.9",
-    "async": "1.5.2",
-    "chalk": "1.1.3",
-    "code-point-at": "1.1.0",
-    "colors": "1.1.2",
-    "concat-stream": "1.5.0",
-    "debug": "2.6.9",
-    "decamelize": "1.2.0",
-    "es6-map": "0.1.5",
-    "escope": "3.6.0",
-    "espree": "3.5.1",
-    "esprima": "2.7.3",
-    "estraverse": "4.2.0",
-    "esutils": "2.0.2",
-    "eventemitter2": "0.4.14",
-    "execa": "0.7.0",
-    "exit": "0.1.2",
-    "file-sync-cmp": "0.1.1",
-    "flat-cache": "1.3.0",
-    "fs.realpath": "1.0.0",
-    "get-caller-file": "1.0.2",
-    "get-stdin": "4.0.1",
-    "getobject": "0.1.0",
-    "globals": "9.18.0",
-    "graceful-fs": "4.1.11",
-    "grunt-known-options": "1.1.0",
-    "hooker": "0.2.3",
-    "ignore": "3.3.7",
-    "imurmurhash": "0.1.4",
-    "inflight": "1.0.6",
-    "inherits": "2.0.3",
-    "inquirer": "0.12.0",
-    "is-fullwidth-code-point": "1.0.0",
-    "is-my-json-valid": "2.16.1",
-    "is-resolvable": "1.0.0",
-    "isarray": "1.0.0",
-    "isexe": "2.0.0",
-    "json-stable-stringify": "1.0.1",
-    "lcid": "1.0.0",
-    "levn": "0.3.0",
-    "locate-path": "2.0.0",
-    "mem": "1.1.0",
-    "meow": "3.7.0",
-    "minimatch": "3.0.4",
-    "mkdirp": "0.5.1",
-    "normalize-package-data": "2.4.0",
-    "object-assign": "4.1.1",
-    "once": "1.4.0",
-    "optionator": "0.8.2",
-    "parse-json": "2.2.0",
-    "path-is-absolute": "1.0.1",
-    "path-is-inside": "1.0.2",
-    "pify": "2.3.0",
-    "pluralize": "1.2.1",
-    "progress": "1.1.8",
-    "require-directory": "2.1.1",
-    "require-main-filename": "1.0.1",
-    "require-uncached": "1.0.3",
-    "rimraf": "2.2.8",
-    "set-blocking": "2.0.0",
-    "strip-ansi": "3.0.1",
-    "strip-bom": "3.0.0",
-    "strip-json-comments": "1.0.4",
-    "table": "3.8.3",
-    "text-table": "0.2.0",
-    "user-home": "2.0.0",
-    "wrap-ansi": "2.1.0",
-    "y18n": "3.2.1"
-  },
+  "_phantomChildren": {},
   "_requested": {
     "name": "@okta/i18n",
-    "raw": "@okta/i18n@37.2.0",
-    "rawSpec": "37.2.0",
+    "raw": "@okta/i18n@37.3.0",
+    "rawSpec": "37.3.0",
     "scope": "@okta",
-    "spec": "37.2.0",
+    "spec": "37.3.0",
     "type": "version"
   },
   "_requiredBy": [
-    "/"
+    "#USER"
```

Comment:
I think this should be ```/```

Reply:
ok, but why did "npm install -E @okta/i18n@37.3.0" do this?